css/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #ffe4e1;
+}
+
+header {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 0;
+  padding: 20px;
+}
+
+.logo {
+  font-family: sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.main-page {
+  display: flex;
+  flex-flow: column nowrap;
+  padding: 20px 1.2rem;
+  margin: 0;
+  background-image: url(../images/bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.main-page-h1 {
+  font-size: 36px;
+  font-weight: 700;
+}
```

Comment:
- [ ] Please kindly match the margins and paddings of your design with Figma design,  eg., on your design there is a much more space between the `h1` and the `p` tags: 

## Yours 

![image](https://user-images.githubusercontent.com/77038610/143383739-0991d70d-1290-44c9-8700-cabb95af88be.png)

## Original

![image](https://user-images.githubusercontent.com/77038610/143383784-48e124bc-fa99-4fb9-a111-ad96c8c3fb16.png)

- [ ] Please fix margins and paddings on other sections as well, as I have recommended, you can set `body{ width to 375px }` and make sure that everything matches 👍🏻

Reply:
Shoira, shuni hech qila olmayapman, 
orasidagi joini kamaitra olmayapman,
nima qilsam bo'ladi
iltimos